cmd/create/execution_test.go

```diff
@@ -19,6 +19,7 @@ import (
 func createExecutionSetup() {
 	ctx = testutils.Ctx
 	mockClient = testutils.MockClient
+	executionConfig = &ExecutionConfig{}
```

Comment:
added this to avoid having leftovers in the execution config that will mess up with the tests